net-libs/tox/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<use>
+		<flag name="av">Adds support for audio and video.</flag>
+		<flag name="log">Enables logging, useful for debugging.
+			(default log level: DEBUG)</flag>
+		<flag name="log-trace">Set log level to TRACE (only choose one log-*,
+			needs log enabled)
+		</flag>
+		<flag name="log-debug">Set log level to DEBUG (only choose one log-*,
+			needs log enabled)
+		</flag>
+		<flag name="log-info">Set log level to INFO (only choose one log-*,
+			needs log enabled)
+		</flag>
+		<flag name="log-warn">Set log level to WARNING (only choose one log-*,
+			needs log enabled)
+		</flag>
+		<flag name="log-error">Set log level to ERROR (only choose one log-*,
+			needs log enabled)
+		</flag>
```

Comment:
I believe log level should be really set via configuration, not at build time and certainly not via a huge set of USE flags. Long story short, this is something most of users don't want to be concerned with. Not that I'm going to force you to fix it at some sane value.

However, if you say 'only choose one log', then I'd expect `REQUIRED_USE` preventing people from selecting more than one.